test/matlab_experimental/ctTestSamples.m

```diff
@@ -0,0 +1,48 @@
+classdef ctTestSamples < ctTestCase
```

Comment:
What's the difference between this file and the previous one? They both appear to run the samples for testing purposes